modules/repository/create.go

```diff
@@ -330,6 +332,26 @@ func UpdateRepoSize(ctx context.Context, repo *repo_model.Repository) error {
 		return fmt.Errorf("updateSize: GetLFSMetaObjects: %w", err)
 	}
 
+	if setting.EnableSizeLimit && repo.GetActualSizeLimit() > 0 && size+lfsSize > repo.GetActualSizeLimit() {
```

Comment:
The logic is not right. The function should just update Repository size.

Reply:
Raised https://github.com/DmitryFrolovTri/gitea-limit-repo-size/issues/16 to address

Reply:
@lunny I have revereted the issue by this commit https://github.com/go-gitea/gitea/pull/21820/commits/22bb76c9c646b3082031d8f5ee6b94a68a7b7e72

The tests also pass on my local

Reply:
I have resolved. please "unresolve" if not ok.